dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/message/LengthFieldStreamingDecoder.java

```diff
@@ -20,7 +20,6 @@
 import org.apache.dubbo.remoting.http12.exception.DecodeException;
 
 import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
 import java.io.IOException;
```

Comment:
**Suggestion**: Removal of unused imports and variables should be double-checked for potential impacts on functionality, especially in encoding and decoding processes. <br> **Code Suggestion**: 
 ```
 - import java.io.ByteArrayOutputStream;
 - private InputStream dataHeader = new ByteArrayInputStream(new byte[0]);
 + // Removal of ByteArrayOutputStream usage
 + // Simplification of stream handling logic
 ```